std_msgs/Header       header
uint32                image_width
uint32                image_height
uint32                focal_lenth_x
uint32                focal_lenth_y
uint32[]              rec_height
uint32[]              rec_width
float32[]             rotation_angle
geometry_msgs/Point   tar
uint32 rectangle_activate
uint32 rectangle_terminate
